### ISSUE TITLE
E202,E203,E222 not detected on multiline string

### ISSUE THREAD

**Issue by spaceone** [hhatto/autopep8#260]
```
$ printf 'foo =  """bar\nbaz"""' | autopep8 -
foo =  """bar
baz"""
```

autopep8 0.9.1

**Comment by myint** [comment]
I think you should check if `pycodestyle` detects this. `autopep8` uses `pycodestyle` to detect problems. So if it doesn't detect it, `autopep8` probably won't fix it.

This applies to #256 and #257 as well.

**Comment by myint** [comment]
I just noticed your issue in the `pycodestyle` repository. So I guess you already figured it out before I commented.

**Comment by spaceone** [comment]
Yes, I tried to fix it myself but the lines weren't ever reached so I saw that it uses pep8 to check the errors. This issue is not caused by pep8 as you can see in:
```
$ printf 'foo =  """bar\nbaz"""\n' | pep8 -
stdin:1:6: E222 multiple spaces after operator
```

**Comment by spaceone** [comment]
```
                cur.execute("""
        INSERT OR REPLACE INTO '%s' (key,value)
            VALUES (  ?, ?
        );""" % section , [option, value] )
```

**Comment by spaceone** [comment]
E203 ```""" % {'urls' : urls,```

**Comment by hhatto** [comment]
fix in latest master.

It is scheduled to be released in version 1.3.
